### PR TITLE
Renamed Node Lable and Tooltip Update

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

This PR is a follow-up PR of https://github.com/DynamoDS/Dynamo/pull/10365. 

This PR cleans up un-useful code and added localization support for new resource strings involved.
New localized strings: 

- `Original node name: `
- `Description: `

The new UI still work as expected:
image

The two new unit tests still pass:
image

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers
@DynamoDS/dynamo 

### FYIs
@Amoursol

### CODE REVIEW

**Comment by QilongTang** [comment]
Running self CI

File: test/core/originalNodeNameTests/NestedNodeCustomNode.dyf [hunk 1]
```diff
@@ -0,0 +1,164 @@
+{
```
Comment (mmisol) [inline test/core/originalNodeNameTests/NestedNodeCustomNode.dyf]: This file does not seem to be used.

**Comment by QilongTang** [comment]
@mmisol The UX group did review this and saying this was the intended design

File: test/core/originalNodeNameTests/NestedNodeCustomNode.dyf (hunk 1 above)
Comment (QilongTang) [reply test/core/originalNodeNameTests/NestedNodeCustomNode.dyf]: @mmisol If you search for the guid you can see there is a dependency on this dyf

Comment (mmisol) [reply test/core/originalNodeNameTests/NestedNodeCustomNode.dyf]: Sorry, I assumed this file needed to be referenced, by name, somewhere in the tests. Just to satisfy my curiosity, could you explain how this file is being used? Thanks

Comment (QilongTang) [reply test/core/originalNodeNameTests/NestedNodeCustomNode.dyf]: @mmisol 
Sure You can see the origianNodeNameCustomNode.dyn has a property called
```Json
  "Dependencies": [
    "6f65de85-d71f-46e1-b995-dfdeba2adbe3"
  ],
```
This guid is the same guid of this dyf, so you can tell that DYN is actually referencing this dyf. placed this custom node and renamed this node

**Pull request merged**